endpoint_logger_response.py

```
@@ -0,0 +1,81 @@
+        }
+
+        # Store the request by its unique flow ID (this helps with pairing later)
+        self.requests_log[flow.id] = request.url
+
+        if request.url:
+            try:
+                with open(self.log_file, "a") as f:
+                    json.dump(log_entry, f)
+                    f.write("\n")
+                    f.flush()  # Force immediate write
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)